### ISSUE TITLE
noResolution directive not detected on named imports

### ISSUE THREAD

**Issue by hazzard993** [TypeScriptToLua/TypeScriptToLua#635]
```ts
/** @noResolution */
declare module "module" {
    export const x: number;
}
```

```ts
import { x } from "module";
x;
```

The `@noResolution` directive in this instance is not picked up so TypeScriptToLua will attempt to resolve the path to `"module"` which will change based off the file using this import.
